src/OctoshiftCLI.Tests/gei/Handlers/MigrateRepoCommandHandlerTests.cs

```diff
@@ -356,10 +360,12 @@ public async Task Happy_Path_GithubSource_Ghes()
             _mockSourceGithubApi.Setup(x => x.GetArchiveMigrationUrl(SOURCE_ORG, gitArchiveId).Result).Returns(gitArchiveUrl);
             _mockSourceGithubApi.Setup(x => x.GetArchiveMigrationUrl(SOURCE_ORG, metadataArchiveId).Result).Returns(metadataArchiveUrl);
 
-            _mockHttpDownloadService.Setup(x => x.DownloadToBytes(gitArchiveUrl).Result).Returns(gitArchiveContent);
-            _mockHttpDownloadService.Setup(x => x.DownloadToBytes(metadataArchiveUrl).Result).Returns(metadataArchiveContent);
-            _mockAzureApi.Setup(x => x.UploadToBlob(It.IsAny<string>(), gitArchiveContent).Result).Returns(authenticatedGitArchiveUrl);
-            _mockAzureApi.Setup(x => x.UploadToBlob(It.IsAny<string>(), metadataArchiveContent).Result).Returns(authenticatedMetadataArchiveUrl);
+            _mockHttpDownloadService.Setup(x => x.DownloadToFile(gitArchiveUrl, gitArchiveFilePath));
+            _mockHttpDownloadService.Setup(x => x.DownloadToFile(metadataArchiveUrl, metadataArchiveFilePath));
+
+            _mockFileSystemProvider.Setup(x => x.OpenRead(gitArchiveFilePath)).Returns(It.IsAny<FileStream>());
```

Comment:
these setups can safely be removed the reason is where we download the file is an implementation detail, so the `migrate-repo` command shouldn't care about the location of the downloaded archive.

Reply:
Addressed in 7d436e04071bf868990e1d32d6276668c1db1a35 :tada: